ConsumerApi.Sdk/Endpoints/Files/Types/CreateFileRequest.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+
+namespace Backbone.ConsumerApi.Sdk.Endpoints.Files.Types;
+
+public class CreateFileRequest
+{
+    public required FileData Content { get; set; }
+    public required string Owner { get; set; }
+    public required string OwnerSignature { get; set; }
+
+    public required string CipherHash { get; set; }
+
+    public required DateTime ExpiresAt { get; set; }
+
+    public required string EncryptedProperties { get; set; }
+
+    public MultipartFormDataContent ToMultipartContent()
+    {
+        MultipartFormDataContent content = [];
+
+        if (Content.IsRealFile) content.Add(new StreamContent(Content.ReadStream), "content", Content.FilePath);
+        else content.Add(new StreamContent(Content.ReadStream), "content");
+
+        content.Add(new StringContent(Owner), "owner");
+        content.Add(new StringContent(OwnerSignature), "ownerSignature");
+        content.Add(new StringContent(CipherHash), "cipherHash");
+        content.Add(new StringContent(ExpiresAt.ToString(CultureInfo.InvariantCulture)), "expiresAt"); //TODO: Specify Date
```

Comment:
What's that TODO comment?

Reply:
I wasn't sure how to format the date, so that it is consistent with the "standard" format of the DateTime class

Reply:
There is an extension method in `Backbone.Tooling.Extensions` called `ToUniversalString`. You can reference the project and use it.

Reply:
Thanks

Reply:
👍